### DISCUSSION TITLE
Welcome to im2latex Discussions!

### DISCUSSION THREAD

**Discussion by untrix** [untrix/im2latex#24] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by KoushikMSD** [comment]
I was going through all the steps with the pre-trained model.I downloaded dataset5 from the the project website,which contains **"data_props.pkl"** file, but when I use this to run
`python run.py -a 0.0001 -e -1 -b 25 -v -1 -i 2 --r-lambda 0.00005 --raw-data-folder C:\Users\Koushik/predictions\inferencing_50 --test --save-all-eval --restore C:\Users\Koushik/predictions\logdir --image-folder C:\Users\Koushik/predictions/formula_images` as described in the notebook **predict.ipynb** I got an error - **raise ParamsValueError('%s is not a valid value of property %s'%(val, name))
commons.dl_commons.ParamsValueError: 338 is not a valid value of property CTCBlankTokenID**
Can you please guide me to get rid of that ?
